docs/vite.config.js

```diff
@@ -3,5 +3,6 @@ export default {
       fsServe: {
         root: '../' 
       }
-    }
+    },
+    optimizeDeps: { include: ['moment-mini', '@braintree/sanitize-url', 'dagre', 'dagre-d3', 'graphlib'] }
```

Comment:
Workaround due to some upstream issues between `vitepress-plugin-mermaid` & `@mermaid-js/mermaid`